syntax = "proto3";
import "grpc_proto_files/CoSiMaMessages.proto";

package CoSiMa.rpc;

service CARLAInterface {
	rpc SetConfig(CarlaConfig) returns (Int32) {}
}

message CarlaConfig {
	string carla_host = 1;
	int32 carla_port = 2;
	double transaction_timeout = 3;
	double delta_seconds = 4;
}
